src/frontend/scripts/app.js

```diff
@@ -1,12 +1,34 @@
 $(document).ready(() => {
-  $('.hamburger').click(() => $('.sidebar-desktop').removeClass('sidebar-mobile'));
-  $('.content-wrapper').click(() => $('.sidebar-desktop').addClass('sidebar-mobile'));
+  const $sidebar = $('.sidebar')
+  $('.hamburger').click(() => $sidebar.toggleClass('sidebar-show'))
+  $('.content-wrapper section').click(() => $sidebar.removeClass('sidebar-show'))
 
-  const $pickadate = $('.pickadate');
+  const $pickadate = $('.pickadate')
   if ($pickadate) {
     $pickadate.pickadate({
       format: 'yyyy-mm-dd',
       firstDay: 1,
     })
-  };
-});
\ No newline at end of file
+  }
+
+  const $dropdownToggle = $('.dropdown-toggle')
+  const toggleMenu = ($element) => {
+    $element.parent().siblings('.dropdown-list').slideToggle()
+    $element.toggleClass('icomoon-dropdown-open').toggleClass('icomoon-dropdown-close')
+  }
+  if ($dropdownToggle) {
+    $dropdownToggle.on('click', (event) => {
+      const $target = $(event.target)
+      toggleMenu($target)
+      const status = $target.hasClass('icomoon-dropdown-open') ? 'open' : 'close'
+      localStorage.setItem($target.data('menu'), status)
+    });
+
+    $dropdownToggle.each((index, element) => {
+      const $element = $(element)
+      if (localStorage.getItem($element.data('menu')) === 'close') {
+        toggleMenu($element)
```

Comment:
When I toggle all the menu items (that they are all hidden) - and refresh the page -> on load there is an animation of hiding - expected result: they should be hidden without any animation after refreshing the page.